frankenstein/notes.md

```diff
@@ -0,0 +1,40 @@
+= Project Frankenstein
```

Comment:
Not sure if I should leave any nits in this notes file, but it's not Markdown-formatted although it ends with `.md`.